src/java.management/share/classes/javax/management/modelmbean/InvalidTargetObjectTypeException.java

```diff
@@ -46,61 +46,13 @@
  *
  * @since 1.5
  */
-@SuppressWarnings("serial")  // serialVersionUID not constant
 public class InvalidTargetObjectTypeException  extends Exception
 {
-
-    // Serialization compatibility stuff:
-    // Two serial forms are supported in this class. The selected form depends
-    // on system property "jmx.serial.form":
-    //  - "1.0" for JMX 1.0
-    //  - any other value for JMX 1.1 and higher
-    //
-    // Serial version for old serial form
-    private static final long oldSerialVersionUID = 3711724570458346634L;
-    //
-    // Serial version for new serial form
-    private static final long newSerialVersionUID = 1190536278266811217L;
-    //
-    // Serializable fields in old serial form
-    private static final ObjectStreamField[] oldSerialPersistentFields =
-    {
-      new ObjectStreamField("msgStr", String.class),
-      new ObjectStreamField("relatedExcept", Exception.class)
-    };
-    //
-    // Serializable fields in new serial form
-    private static final ObjectStreamField[] newSerialPersistentFields =
+    private static final long serialVersionUID = 1190536278266811217L;
+    private static final ObjectStreamField[] serialPersistentFields =
```

Comment:
The API doc comment for the serialized fields dhould be preserved.
```suggestion
    /**
     * @serialField exception Exception Encapsulated {@link Exception}
     */
    private static final ObjectStreamField[] serialPersistentFields =
```